test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1

```diff
@@ -70,6 +70,24 @@ Describe "Stream writer tests" -Tags "CI" {
     }
 
     Context "Write-Information cmdlet" {
+        BeforeAll {
+            $rs = [runspacefactory]::Createrunspace()
+            $rs.open()
+            $ps = [powershell]::Create()
+            $ps.Runspace = $rs
+            $ps = [powershell]::Create()
```

Comment:
You created 2 PowerShell instances here :)
`$ps = [powershell]::Create()` should be enough here. You don't need to explicitly create the runspace because `$ps.Invoke()` will automatically create a new runspace using `[runspacefactory]::Createrunspace()` is `$ps.Runspace` is null.